lib/renderer.js

```diff
@@ -132,7 +132,11 @@ module.exports = function(data, options) {
     }
   });
 
+  // exec filter to extend renderer.
+  const renderer = new Renderer();
+  this.execFilterSync('marked:renderer', renderer, {context: this});
```

Comment:
I assume `execFilter()` is not used because marked is not async?

Reply:
@curbengh `marked` support async, but in `hexo-renderer-marked`, no.

Reply:
is it possible for this plugin?

Reply:
https://github.com/hexojs/hexo-renderer-marked/pull/132

Reply:
> I assume `execFilter()` is not used because marked is not async?

I tried `execFilter()`, but fail. This is beyond my ability :joy: . @curbengh good job.